packages/vue/src/components/organisms/SfGroupedProduct/SfGroupedProduct.js

```diff
@@ -0,0 +1,61 @@
+import Vue from "vue";
+import Glide from "@glidejs/glide";
+import SfGroupedProductItem from "./_internal/SfGroupedProductItem.vue";
+
+Vue.component("SfGroupedProductItem", SfGroupedProductItem);
+
+export default {
```

Comment:
shouldn't we pass glide options as prop like on other components?

Reply:
sure :)